From 6b3b3340476bb83f2e228a3de2491dd0f9772e5f Mon Sep 17 00:00:00 2001
From: Takeshi Kihara <takeshi.kihara.df@renesas.com>
Date: Wed, 5 Sep 2018 20:28:24 +0530
Subject: [PATCH 479/909] arm64: dts: r8a77990-ebisu: Remove renesas,
 no-ether-link property

commit 69360bc0b67fda689885bff813c9fb94d4f9e2df from
git://git.kernel.org/pub/scm/linux/kernel/git/horms/renesas-bsp.git

The present change is a bug fix for AVB link iteratively up/down.
This patch is based on commit 87c059e9c39d ("arm64: dts: renesas:
salvator-x: Remove renesas, no-ether-link property").

Signed-off-by: Takeshi Kihara <takeshi.kihara.df@renesas.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/renesas/r8a77990-ebisu.dts | 1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/arm64/boot/dts/renesas/r8a77990-ebisu.dts b/arch/arm64/boot/dts/renesas/r8a77990-ebisu.dts
index 3aa612e1656b..9a2dad4cdab2 100644
--- a/arch/arm64/boot/dts/renesas/r8a77990-ebisu.dts
+++ b/arch/arm64/boot/dts/renesas/r8a77990-ebisu.dts
@@ -138,7 +138,6 @@
 &avb {
 	pinctrl-0 = <&avb_pins>;
 	pinctrl-names = "default";
-	renesas,no-ether-link;
 	phy-handle = <&phy0>;
 	phy-mode = "rgmii";
 	status = "okay";
-- 
2.17.1

